traits/autoscalertrait/charts/autoscalertrait/values.yaml

```
@@ -0,0 +1,79 @@
+# Default values for autoscalertrait.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/oam-dev/catalog/autoscalertrait
+  pullPolicy: IfNotPresent
```

only image with a revision tag suggest using `IfNotPresent`

```suggestion
  pullPolicy: Always
```